test/powershell/Language/Classes/Scripting.Classes.Modules.Tests.ps1

```diff
@@ -76,9 +76,9 @@ Describe 'Module reloading with Class definition' -Tags "CI" {
 
     BeforeAll {
         Set-Content -Path TestDrive:\TestModule.psm1 -Value @'
-$passedArgs = $args
-class Root { $passedIn = $passedArgs }
-function Get-PassedArgsRoot { [Root]::new().passedIn }
+$passedArgs = $args
+class Root { $passedIn = $passedArgs }
+function Get-PassedArgsRoot { [Root]::new().passedIn }

```

Comment:
Didn't change anything here as PSModulePath isn't here.  Not sure why this is showing up in the PR as a change.